docs/tutorials/testing_with_control_center.rst

```diff
@@ -3,33 +3,27 @@
 
 .. _control_center_tutorial:
 
-*********************************************************
-Getting Started with Speed Motors Using IQ Control Center
-*********************************************************
+************************************************************
+Getting Started with Speed Modules Using IQ Control Center
+************************************************************
 
-The easiest way to get started with any Vertiq Motor is with the IQ Control Center. 
+The easiest way to get started with any Vertiq module is with the IQ Control Center. 
 The IQ Control Center is a simple application that allows you to configure and test
-a Vertiq Motor from your PC, without the need for a flight controller or specialized hardware.
+a Vertiq module from your PC, without the need for a flight controller or specialized hardware.
 
-This tutorial will take you through the process of getting started with a motor using
+This tutorial will take you through the process of getting started with a module using
 the IQ Control Center. It will outline how to setup the necessary hardware and software,
 provide a brief introduction to the Control Center, and teach you how to peform some basic
-tests on the motor with the Control Center.
+tests on the module with the Control Center.
 
-**This tutorial is applicable for both the Vertiq 2306 and the Vertiq 8108, but only when 
-using the Speed firmware. If you are using the position firmware, some of the parameters discussed here
-will not be available.**
+**This tutorial was created using a Vertiq 8108 module, on firmware version 0.0.7 and with Control Center version 1.3.4. However, this tutorial is applicable for any Vertiq module when using
+speed firmware. If you are using the position firmware, some of the parameters discussed here will not be available.**
 
-Motor Datasheets
+Module Datasheets
 ===================
-Before trying to use the motors, it is advised to read through their datasheets to
-get a better sense of what they can do and how the hardware is setup. The datasheets
-for all Vertiq modules can be found under the `Support <https://www.vertiq.co>`_
-section of our website, and individual links are provided below for convenience:
-
-* `Vertiq 2306 2200KV <https://static1.squarespace.com/static/62981f8ddf7dc07b7eed3f3f/t/63063670f72c060db8879052/1661351539244/Vertiq_23-06_2200kv_module_datasheet.pdf>`_
-* `Vertiq 2306 220KV <https://static1.squarespace.com/static/62981f8ddf7dc07b7eed3f3f/t/63066cb62767660deec6dadd/1661365431560/Vertiq_23-06_220kv_module_datasheet.pdf>`_
-* `Vertiq 8108 150KV <https://static1.squarespace.com/static/62981f8ddf7dc07b7eed3f3f/t/6306367b2b9a912706a32f22/1661351553848/Vertiq_81-08_150kv_module_datasheet.pdf>`_
+Before trying to use the modules, it is recommended to read through their datasheets to
```

Comment:
Before trying to use your module, we recommend that you read through its datasheet ...